CONTRIBUTING.md

```diff
@@ -171,18 +175,18 @@ automatically build your code, restart the backend and refresh the frontend
    ```
    pnpm dev
    ```
-1. Hack, hack, hack
-1. Check if everything still runs in production mode
+2. Hack, hack, hack
```

Comment:
Suggest we keep the repeated `1.` as it makes it easier to edit the list with more/fewer steps in future. If anything, all the numbered lists should do this.

Reply:
Not sure I understand or agree with this logic (perhaps something that is in practice and just haven't come across it till now), but I've gone ahead and reverted the change.